features/git-ship/current_branch/on_feature_branch_without_open_changes.feature

```diff
@@ -38,8 +38,10 @@ Feature: git ship: shipping the current feature branch
   Scenario: feature branch with non-pulled updates in the repo
     Given I have a feature branch named "feature"
     And the following commit exists in my repository
-      | BRANCH  | LOCATION | FILE NAME    | FILE CONTENT    |
-      | feature | remote   | feature_file | feature content |
+      | BRANCH  | LOCATION         | FILE NAME    | FILE CONTENT          |
```

Comment:
Okay I will do a separate PR implementing remote commits as another git repo for the local user. Will also try to lazily create repos as we need them